bucket/cv4pve-pepper.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.4.0",
+    "description": "Launching SPICE remote-viewer to access VM running on Proxmox VE.",
+    "homepage": "https://github.com/Corsinvest/cv4pve-pepper",
+    "license": "GPL v3",
+    "architecture": {
```

Comment:
https://github.com/Corsinvest/cv4pve-pepper#run

Reply:
What is this?

Reply:
That there are clearly some dependencies??

![image](https://user-images.githubusercontent.com/13260377/128568545-8ef8c7a1-2de8-48cd-9050-696b5592b997.png)

Reply:
Ok, I have added the `"depends": "extras/virt-viewer"` as its dependency.